includes/admin/OrderSettings.php

```diff
@@ -448,7 +447,7 @@ private function setInsuranceData(): void
         $carrier             = ConsignmentFactory::createByCarrierName($this->carrier);
         $amountPossibilities = $carrier->getInsurancePossibilities();
 
-        if ($insuranceFromDeliveryOptions && $insuranceFromDeliveryOptions >= $amountPossibilities[self::FIRST_INSURANCE]) {
+        if ($insuranceFromDeliveryOptions && $insuranceFromDeliveryOptions >= array_shift($amountPossibilities)) {
```

Comment:
Ik hou niet zo van dit soort functies, maar ik heb dan liever `reset()` in plaats van deze. `array_shift()` past de array aan, en als je dit zo leest weet je niet direct dat daar dan toevallig verder niks mee gedaan wordt. `reset()` is iets minder verwarrend.